projects/1.16/assets/agricraft/agricraft/lang/zh_cn.json

```
@@ -0,0 +1,847 @@
+{
+  "itemGroup.agricraft": "农业工艺",
+  "itemGroup.agricraft_seeds": "农业工艺丨种子",
+
+  "block.agricraft.crop_sticks_wood": "木质作物架",
+  "block.agricraft.crop_sticks_iron": "铁质作物架",
+  "block.agricraft.crop_sticks_obsidian": "黑曜石作物架",
+  "block.agricraft.seed_analyzer": "种子分析仪",
+  "block.agricraft.irrigation_tank": "木质水箱",
+  "block.agricraft.irrigation_channel": "木质灌溉水道",
+  "block.agricraft.irrigation_channel_hollow": "木质灌溉水道（方块）",
+  "block.agricraft.sprinkler": "洒水器",
+  "block.agricraft.grate": "木格栅",
+  "block.agricraft.greenhouse_air": "空气",
+
+  "item.agricraft.debugger": "漏洞测试器",
+
+  "item.agricraft.agri_clipper": "剪枝器",
+  "item.agricraft.agri_journal": "农业研究笔记",
+  "item.agricraft.agri_magnifying_glass": "放大镜",
+  "item.agricraft.agri_rake_iron": "铁耙",
+  "item.agricraft.agri_rake_wood": "木耙",
+  "item.agricraft.agri_seed_bag": "精确的（？）的种子分类袋",
+  "item.agricraft.agri_seed_bag_inactive": "普通的皮包",
+  "item.agricraft.agri_trowel": "园丁铲",
+  "item.agricraft.agri_channel_valve": "灌溉水槽阀门",
+
+  "item.agricraft.agri_nugget_coal": "煤炭碎块",
+  "item.agricraft.agri_nugget_diamond": "钻石粒",
+  "item.agricraft.agri_nugget_emerald": "绿宝石粒",
```

```suggestion
  "item.agricraft.agri_nugget_diamond": "钻石粒",
  "item.agricraft.agri_nugget_emerald": "绿宝石粒",
```
原文Shard
但key里写的nugget，奇怪

---

那就按Shard翻译吧